ios/FluentUI/Vnext/PersonaButton/MSFPersonaButtonTokens.generated.swift

```diff
@@ -42,60 +42,60 @@ extension FluentUIStyle {
 			return mainProxy().Colors.Background.neutral1
 		}
 
-		// MARK: - labelColor 
-		open var labelColor: UIColor {
-			return mainProxy().Colors.Foreground.neutral1
-		}
-
-		// MARK: - labelFont
-		open var labelFont: labelFontAppearanceProxy {
-			return labelFontAppearanceProxy(proxy: mainProxy)
+		// MARK: - horizontalAvatarPadding
+		open var horizontalAvatarPadding: horizontalAvatarPaddingAppearanceProxy {
+			return horizontalAvatarPaddingAppearanceProxy(proxy: mainProxy)
 		}
-		open class labelFontAppearanceProxy {
+		open class horizontalAvatarPaddingAppearanceProxy {
 			public let mainProxy: () -> FluentUIStyle
 			public init(proxy: @escaping () -> FluentUIStyle) {
 				self.mainProxy = proxy
 			}
 
 			// MARK: - large 
-			open var large: UIFont {
-				return mainProxy().Typography.subheadline
+			open var large: CGFloat {
+				return mainProxy().Spacing.xSmall
 			}
 
 			// MARK: - small 
-			open var small: UIFont {
-				return mainProxy().Typography.caption1
+			open var small: CGFloat {
+				return mainProxy().Spacing.medium
```

Comment:
hmmm its interesting how smaller carousel has more larger spacing...

Reply:
Very interesting, but I think it's because we still need almost the same amount of space for text. The avatar is much smaller, but the text is only a little smaller, so the gap has to get bigger to help the label stay readable.